derivers.lua

```diff
@@ -143,8 +237,187 @@ local is = {
 			assert(compiled, message)
 			idx["is_" .. vname] = compiled()
 		end
+
+		t.derived_is = true
+	end,
+	foreign = function()
+		error("can't derive is for a foreign type")
```

Comment:
should edit this to say "can't derive is() for a foreign type" to make it more clear

Reply:
resolved in #95